### PR TITLE
Ac/firmware version

### PR DESCRIPTION
**Pull request by eljuguetero**

### CODE REVIEW

File: demos/CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,8 @@ ADM_EXTRA_WARNINGS()
 
 find_package(Threads)
 
+find_library (FFTW_LIBRARIES NAMES fftw3f )
```
Comment (kdpatino) [inline demos/CMakeLists.txt]: Remove fft libraries, not used yet.

**Comment by kdpatino** [comment]
Be sure just add firmaware id not beamforming files

**Comment by brianofrokk3r** [comment]
@eljuguetero you just opened up another PR for versioning here... on a new branch. What would you like to do with this? Reject?

**Comment by eljuguetero** [comment]
Please reject this PR, accidentally, I combined beamforming commits  with versioning

**Pull request closed without merging**